Scheduler/observing_blocks_maintel/BLOCK-T428.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "BLOCK-T428",
+  "program": "BLOCK-T428",
+  "constraints": [],
+  "scripts": [
+    {
+      "name": "maintel/offset_mtcs.py",
+      "standard": true,
+      "parameters": {
+        "offset_azel": {
```

Comment:
I think you will want to use an offset_xy for this one since you are trying to move the star straight down the field of view. The offset_xy cmd operated in the camera plane, whereas the offset_azel is in the telescope plane.